src/tactic/ui/tools/trigger_wdg.py

```diff
@@ -2119,55 +2130,57 @@ def execute(my):
                 trigger.commit()
                 trigger_code = trigger.get_value("code")
 
+        search_type = my.kwargs.get("search_type")
+        print search_type
 
-        # get the script path
+        # Get the script path or script
         script_path = my.kwargs.get("script_path")
-
-        # get some data
         script = my.kwargs.get("script")
-
-        search_type = my.kwargs.get("search_type")
-
-        # update the trigger
-        trigger.set_value("code", trigger_code)
-        trigger.set_value("script_path", script_path)
-
-        src_status = my.kwargs.get("src_status")
-        if src_status:
-            data = {
-                'src_status': src_status
-            }
-
-            data = jsondumps(data)
-            trigger.set_value("data", data)
-
-        trigger.commit()
-
-        # get the custom script
+      
+        if not script_path:
+            script_path = trigger.get_value("script_path")
```

Comment:
is this a real trigger sobj here as opposed to notification_sobj? cuz you are not using no_exception=True here.